### ISSUE TITLE
Does not work with files ending with .blade.php

### ISSUE THREAD

**Issue by twocatszerokarma** [wcwhitehead/bootstrap-3-snippets#22]
This extension works fine with .html and .php files, just not with laravel blade (.blade.php) files.
Don't think it supports Twig either...

**Comment by HombreSensato** [comment]
Is there any solution for this?

**Comment by jericopulvera** [comment]
does not work in .php
please fix!

**Comment by parasbansal** [comment]
Any solution?

**Comment by HackerFinn** [comment]
I would like to see this as well. I am pretty sure it worked at one point, but I may be mistaken.

**Comment by wcwhitehead** [comment]
Added support for .blade

**Comment by wcwhitehead** [comment]
Reopening. See this thread: https://github.com/wcwhitehead/bootstrap-3-snippets/issues/24

**Comment by Shoukat-bjr** [comment]
Any solution.....????

**Comment by wcwhitehead** [comment]
@Shoukat-bjr Nope. As far as I can tell, there isn't a way to conditionally include snippets if a language extension has been installed. If I include support for blade files, users who do not have a blade extension installed will get an error or a warning message.

**Comment by MaverickMartyn** [comment]
I have seen this done in other extensions, so surely it must be possible.

**Comment by wcwhitehead** [comment]
@HackerFinn Can you point me to one that does? Remember, it needs to be a snippet extension.

**Comment by yamenarahman** [comment]
@wcwhitehead bootstrap4 does. https://github.com/thekalinga/bootstrap4-vscode

**Comment by wcwhitehead** [comment]
@yamenarahman It doesn't look to see if the blade extension is installed and then makes the snippets available -- it just makes it available to blade files. This is what I had originally implemented, but it creates warning messages for anyone who does not have blade support installed.

**Comment by yamenarahman** [comment]
@wcwhitehead You said you want us to point to a package that works fine and we do, now what will you do to remove this pain ?

**Comment by strongui** [comment]
Is there any progress on this? I think there should be a setting for this plugin such as emmet: 

```
"emmet.includeLanguages": {
    "velocity": "html"
  },
```

**Comment by mityukov** [comment]
Please just make it available for .blade.php via an option. There, near that option, you can add whatever disclaimer you'd feel appropriate (like: "please make sure you have blade extension installed before enabling this option", etc.)
